src/Cronarchy.php

```diff
@@ -61,7 +61,7 @@ class Cronarchy
      *
      * @since [*next-version*]
      *
-     * @param string     $instanceId The instance ID. Must be unique.
+     * @param int        $instanceId The instance ID. Must be unique.
```

Comment:
Why was this changed to an integer? The instance ID is intended to be a string, used by plugin/theme vendors to identify their instance. Did phpstan recommend this change on account of the name indicating an "ID"?

Reply:
It is contradicting, e.g. line 28: `@var int` ...

Reply:
Done.

Reply:
> Did phpstan recommend this change on account of the name indicating an "ID"?

No! Skynet has not taken over the globe! :)

Reply:
Someday :D